### PR TITLE
runfix: Adapt the enriched profile to the new backend format (WEBAPP-5952)

### PR DESCRIPTION
**Pull request by atomrc**

### CODE REVIEW

File: src/script/components/panel/enrichedFields.js [hunk 1]
```diff
@@ -18,23 +18,42 @@
...
+    this.richProfileRepository
+      .getUserRichProfile(ko.unwrap(userId))
+      .then(richProfile => {
+        if (richProfile.fields) {
+          this.fields(richProfile.fields);
+          onFieldsLoaded(this.fields());
+        }
+      })
+      .catch(() => {
+        // XXX remove me when the backend endpoint is available
```
Comment (ffflorian) [inline src/script/components/panel/enrichedFields.js]:
```suggestion
        // TODO remove me when the backend endpoint is available
```

File: test/unit_tests/user/RichProfileRepositorySpec.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+/*
+ * Wire
+ * Copyright (C) 2018 Wire Swiss GmbH
```
Comment (ffflorian) [inline test/unit_tests/user/RichProfileRepositorySpec.js]:
```suggestion
 * Copyright (C) 2019 Wire Swiss GmbH
```

File: test/unit_tests/user/RichProfileRepositorySpec.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+describe('RichProfileRepository', () => {
+  let richProfileRepository;
+
+  beforeEach(() => {
+    richProfileRepository = resolve(graph.RichProfileRepository);
+  });
+
+  describe('getUserRichProfile', () => {
+    it("fetches the user's rich profile is not in cache", () => {
```
Comment (ffflorian) [inline test/unit_tests/user/RichProfileRepositorySpec.js]: That sentence doesn't make sense :thinking:

File: test/unit_tests/user/RichProfileRepositorySpec.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+      const response = [];
+      spyOn(richProfileRepository.backendClient, 'sendRequest').and.returnValue(Promise.resolve(response));
+
+      return richProfileRepository.getUserRichProfile(userId).then(richProfile => {
+        expect(richProfileRepository.backendClient.sendRequest).toHaveBeenCalledTimes(1);
+        expect(richProfile).toBe(richProfile);
+      });
+    });
+
+    it("fetches the user's rich profile only once and then load it from cache", () => {
```
Comment (ffflorian) [inline test/unit_tests/user/RichProfileRepositorySpec.js]:
```suggestion
    it("fetches the user's rich profile only once and then loads it from cache", () => {
```

File: test/unit_tests/user/RichProfileRepositorySpec.js (hunk 3 above)
Comment (atomrc) [reply test/unit_tests/user/RichProfileRepositorySpec.js]: lol

**Pull request merged**